README.md

```diff
@@ -101,6 +101,15 @@ $ kubectl get no -w
 **Note:-** You should be seeing nodes joining the cluster within less than minutes.
 
 ---
+#### EKS cluster upgrade using new asg file in terraform
+Create a new eks-worker-node-v1.tf file with different name and below changes you have to do for EKS cluster upgrade.
```

Comment:
Can we have this statement corrected? Why would we need to create a new file? The `eks-worker-node-v1.tf` is already created for people to re-use right?

Reply:
Noo.. It will impact to existing nodes , if you modify existing one then all nodes will be terminated immediately. So we have to create new file with different image and new version. and then move the workload to the new nodes.